### PR TITLE
New nuclear ops breaching charges

### PR DESCRIPTION
**Pull request by Momoberry**
## About the PR 
The PR creates a new type of breaching charge exclusively for nuclear emergency. It's coloured red to match the nuke op palette and to differentiate from others. The explosion is a little larger and doesn't avoid collateral damage like the experimental NT version...

Syndicate Charge compared to standard: https://i.gyazo.com/c0c59d27578ba1ab47d548737b42889e.png
Comparison of syndicate to standard charge: https://i.gyazo.com/33b8b3b674fcc924e96d000790cb418c.png
Syndicate Charge destroying propoganda: https://i.gyazo.com/b8e4604dbe71c8bb789fc479048b1895.png
Warning on Placement: https://i.gyazo.com/a9f167a2764052215a0fb32dcb1d1949.png
## Why's this needed? 
It's good to have variants that are visually apparent, and personally I believe the thermite charge doesn't fit the Nuke Ops gameplay very well. This is loud, and creates a bit of smoke as well, making for a very dramatic entry.

## Changelog

```changelog
(u)Momoberry
(+)New Nuclear Operative breaching charges!
```

### CODE REVIEW

**Comment by Sovexe** [comment]
These are probably too strong currently, could seriously hurt your team. Also might be time to look into using the shaped charges code. I would probably use a width of 90 and then just convert the direction into degrees for the angle

**Comment by Momoberry** [comment]
> These are probably too strong currently, could seriously hurt your team. Also might be time to look into using the shaped charges code. I would probably use a width of 90 and then just convert the direction into degrees for the angle

So I did a fair bit of testing on angled explosions and I honestly feel like they're too inconsistent to be for breaching charges. They'd have to be very high power to consistently explode like the general explosions currently. I agree that the power should be lowered after testing and realizing even a 40% explosion prot was still being instantly gibbed. Will do some changes, but I'm not sure about angled explosions, at least for these. Just because they just aren't that consistent with what they destroy. I feel like the situation of it blowing up a portion, ignoring a wall entirely and blowing up behind it would become pretty common which would require multiple uses to get through something

File: code/obj/item/grenades.dm [hunk 1]
```diff
@@ -1271,6 +1271,19 @@ ABSTRACT_TYPE(/obj/item/old_grenade/spawner)
 	stamina_cost = 1
 	stamina_crit_chance = 0
 
+/obj/item/breaching_charge/syndicate
+	name = "syndicate breaching charge"
+	icon = 'icons/obj/items/grenade.dmi'
```
Comment (Sovexe) [inline code/obj/item/grenades.dm:1276]:
```suggestion
```

this is already inherited

**Comment by Momoberry** [comment]
At 50% explosion prot (Commander), they do just under soft crit, around 95. at 40% prot they do around 120 damage, light crit but easily healed, assuming both explosions were directly on your chest. They will gib an unprotected person. The explosion range was changed a bit and the power was overall weakened. Also added flavour description and added a bolded warning when placing charges, hopefully making it more noticeable that you should get out of the way.

**Comment by Tarmunora** [comment]
mechanically: not a fan - these seem significantly less useful as an actual breaching tool without really bringing anything particularly interesting to the table imo. (plus nuclear operatives already have a significantly more dramatic breach-and-entry tool: the RPG)

**Comment by Momoberry** [comment]
> mechanically: not a fan - these seem significantly less useful as an actual breaching tool without really bringing anything particularly interesting to the table imo. (plus nuclear operatives already have a significantly more dramatic breach-and-entry tool: the RPG)

These are replacing thermites, which were honestly not fitting for nuke ops at all. They were a 5x5, silent fireball that lasted for around 8 seconds, stalling nukies and lighting them on fire, which is genuinely one of the worst situations for them. The RPG is a good breaching tool, but it also has uses outside of being a breaching tool, and requires someone dedicating their inventory to it as it's two handed and only stored on the back

These are meant to be more explosive and visually interesting with the smoke that spreads out from the explosion, and also have some potential as suppression since they are fairly short range but strong, delayed explosions. I'm not sure what's better as a breaching tool beyond the RPG, which people can walk in front of and die, which we see a fair bit.

**Pull request closed without merging**